docfx.json

```
@@ -296,6 +299,7 @@
       "webview2-1054-31": { "moniker_range": "webview2-winrt-1.0.1054.31" },
       "webview2-1020-30": { "moniker_range": "webview2-winrt-1.0.1020.30" },
       "webview2-992-28": { "moniker_range": "webview2-winrt-1.0.992.28" },
+	  "webview2-2950-prerelease": { "moniker_range": "webview2-winrt-preview" },
```

```suggestion
      "webview2-2950-prerelease": { "moniker_range": "webview2-winrt-1.0.2950-prerelease" },
```

We should update the moniker.